### ISSUE TITLE
Method to download original data

### ISSUE THREAD

**Issue by jonesse3** [JDACS4C-IMPROVE/IMPROVE#117]
Should we use a bash script to download the original data since we are no longer using candle's function to retrieve files @wilke @priyanka9991?

**Comment by wilke** [comment]
Yes - we should have a script that downloads the original data, and if possible, the original data should reside on our FTP site

**Comment by wilke** [comment]
Similar to scripts/get-benchmarks

**Comment by priyanka9991** [comment]
Yes I agree. Here is something I have for Paccmann. We can adapt it for other models too.
https://github.com/JDACS4C-IMPROVE/Paccmann_MCA/blob/Parsl/model_specific_data_download.sh

**Comment by adpartin** [comment]
**---  Not urgent  ---**
We need to think about a consistent way to do it (e.g., PaccMann) and make sure all model curators implement this.

**Comment by nkoussa** [comment]
Done
